server/command.go

```diff
@@ -98,84 +98,115 @@ func (c *conn) getRaftCluster() (*raftCluster, error) {
 	return cluster, nil
 }
 
-func (c *conn) handleGetMeta(req *pdpb.Request) (*pdpb.Response, error) {
-	request := req.GetGetMeta()
+func (c *conn) handleGetStore(req *pdpb.Request) (*pdpb.Response, error) {
+	request := req.GetGetStore()
 	if request == nil {
-		return nil, errors.Errorf("invalid get meta command, but %v", req)
+		return nil, errors.Errorf("invalid get store command, but %v", req)
 	}
 
 	cluster, err := c.getRaftCluster()
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 
-	resp := &pdpb.GetMetaResponse{
-		MetaType: request.MetaType,
+	storeID := request.GetStoreId()
+	store, err := cluster.GetStore(storeID)
+	if err != nil {
+		return nil, errors.Trace(err)
 	}
+	// Store may be nil, should we return an error instead of none result?
```

Comment:
I think `cluster.GetStore()` guarantees that if `err` is nil, `store` must not be nil.